packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -603,6 +603,9 @@ export function getJobStatisticsHtml(jobStatistics: any) {
     return htmlString
 }
 
+export const getBillingString = (linesOfCode: number): string =>
```

Comment:
Why do we need this function that just wraps the same contents as codeTransformBillingText? Can't we just call it directly?

Reply:
That's true; fixed